homeassistant/components/kodi/browse_media.py

```diff
@@ -194,9 +196,15 @@ def item_payload(item, media_library):
     if thumbnail:
         thumbnail = media_library.thumbnail_url(thumbnail)
 
+    if media_content_type == MEDIA_TYPE_MOVIE and not bool(media_content_id):
```

Comment:
No need to cast to bool, happens automatically when using in an expression.
```suggestion
    if media_content_type == MEDIA_TYPE_MOVIE and not media_content_id:
```